src/UI/templates/js/Item/notification.js

```diff
@@ -503,7 +507,7 @@ il.UI.item = il.UI.item || {};
 			var getNotificationsTriggererIfAny = function(){
 				var $meta_bar = getMetaBarOfItemIfIsInOne();
 				if($meta_bar.length){
-					var $notification_glyph = $meta_bar.find('.il-maincontrols-metabar > .btn-bulky .glyphicon-bell');
+					var $notification_glyph = $meta_bar.find('.btn-bulky .glyphicon-bell');
```

Comment:
This change looks wrong an dangerous. This would detect any .glyphicon-bell as notification trigger. This might break if we would use this glyph in any place other than the meta bar. Better just add the missing li; 
.il-maincontrols-metabar > li > .btn-bulky .glyphicon-bell

Reply:
That's true, though the selector `.il-maincontrols-metabar > li > .btn-bulky .glyphicon-bell` doesn't quite do the trick.